### PR TITLE
improve kubernetes docs

### PR DESCRIPTION
**Pull request by itaysk**

### CODE REVIEW

File: docs/install/prerequisites.md [hunk 1]
```diff
@@ -1,21 +1,13 @@
 # Prerequisites
 
-__Portable (CO:RE) option:__
-* Linux kernel version >= 4.18
-* BTF enabled (_You can manually detect if your environments supports it by checking if the following file exists on your machine: /sys/kernel/btf/vmlinux or consult the following documentation: https://github.com/libbpf/libbpf#bpf-co-re-compile-once--run-everywhere)_
+* Linux kernel version >= 4.18.
+* libc, and the libraries: libelf, zlib.
+* Access to read kernel configuration. This can be in `/proc/config.gz` (auto-mounted in Docker) or `/boot/config-$(uname -r)` (requires mounting in Docker), depending on the Linux distribution.
```
Comment (yanivagman) [inline docs/install/prerequisites.md:5]: This is not a prerequisite for non-CO-RE option

File: docs/install/prerequisites.md [hunk 2]
```diff
@@ -1,21 +1,13 @@
 # Prerequisites
 
-__Portable (CO:RE) option:__
-* Linux kernel version >= 4.18
-* BTF enabled (_You can manually detect if your environments supports it by checking if the following file exists on your machine: /sys/kernel/btf/vmlinux or consult the following documentation: https://github.com/libbpf/libbpf#bpf-co-re-compile-once--run-everywhere)_
+* Linux kernel version >= 4.18.
+* libc, and the libraries: libelf, zlib.
```
Comment (yanivagman) [inline docs/install/prerequisites.md:4]: This is not a requirement for CO-RE option (as the object is already compiled).
Also, it is not a requirement when running from the docker image

Comment (itaysk) [reply docs/install/prerequisites.md:4]: i thought libelf and zlib are runtime requirements of libbpf, no?

Comment (yanivagman) [reply docs/install/prerequisites.md:4]: We are using libbpf.a which statically compiles these libs into it

**Review by yanivagman — APPROVED** [review]

**Review by simar7 — APPROVED** [review]

**Pull request merged**